src/Views/Admin/Pages/templates/setup-page/index.html.php

```diff
@@ -0,0 +1,114 @@
+<div class="wrap" class="give-setup-page">
+
+	<h1 class="wp-heading-inline">
+		<?php echo __( 'Setup GiveWP', 'give' ); ?>
+	</h1>
+
+	<hr class="wp-header-end">
+
+	<!-- Configuration -->
+	<?php
+		echo $this->render_template(
+			'setup-page/section',
+			[
+				'title'    => 'Create your first donation form in minutes',
+				'badge'    => '<span class="badge badge-review">5-10 Minutes</span>',
+				'contents' => $this->render_template(
+					'setup-page/row-item',
+					[
+						'class'       => 'configuration',
+						'icon'        => $this->image( 'configuration@2x.min.png' ),
+						'icon_alt'    => esc_html__( 'First-Time Configuration', 'give' ),
+						'title'       => esc_html__( 'First-Time Configuration', 'give' ),
+						'description' => esc_html__( 'Every fundraising campaign begins with a donation form. Click here to create your first donation form in minutes. Once created you can use it anywhere on your website.', 'give' ),
+						'action'      => '<a href="' . admin_url( '?page=give-onboarding-wizard' ) . '"><span class="screen-reader-text">Configure GiveWP</span><i class="fas fa-chevron-right"></i></a>',
+					]
+				),
+			]
+		);
+		?>
+
+	<!-- Gateways -->
+	<?php
+		echo $this->render_template(
+			'setup-page/section',
+			[
+				'title'    => 'Connect a payment gateway to begin accepting donations',
+				'contents' => [
+					$this->render_template(
+						'setup-page/row-item',
+						[
+							'icon'        => $this->image( 'paypal@2x.min.png' ),
+							'icon_alt'    => esc_html__( 'PayPal', 'give' ),
+							'title'       => esc_html__( 'Connect to PayPal', 'give' ),
+							'description' => esc_html__( 'PayPal is synonymous with nonprofits and online charitable gifts. It’s been the go-to payment merchant in for many of the worlds top NGOs. Accept PayPal, Credit and Debit Cards, and more using PayPal’s Smart Buttons without any added platform fees.', 'give' ),
+							'action'      => '<img src="' . GIVE_PLUGIN_URL . 'assets/dist/images/setup-page/paypal.svg' . '" alt="Connect to PayPal" />',
```

Comment:
Why is the PayPal setup button an image here? It looks fuzzy on the front end (compared to Stripe) and is not accessible via keyboard navigation.

Reply:
@henryholtgeerts I'm using the images as a placeholder to preview design, until the new PayPal gateway is available to integrate.